src/webui/www/private/scripts/dynamicTable.js

```diff
@@ -1225,6 +1225,18 @@ window.qBittorrent.DynamicTable = (function() {
                     if (~state.indexOf('paused'))
```

Comment:
(Not related to this PR)
Using `~` bitwise operator instead of proper logical operator is quite bad here. It should be replaced by properly comparing with `-1` (the not found value).

Reply:
https://github.com/qbittorrent/qBittorrent/pull/11871